ui/src/dashboards/components/variablesControlBar/VariableDropdown.tsx

```diff
@@ -87,7 +87,11 @@ const mstp = (state: AppState, props: OwnProps): StateProps => {
   return {values, selectedValue}
 }
 
-export default connect<StateProps, {}, OwnProps>(
+const mdtp = {
+  onSelectValue: selectVariableValue as any,
```

Comment:
Since it is a thunk, TypeScript sees its type as
```
(dashboardID: string, variableID: string, value: string) => (dispatch: any, getState: GetState) => Promise<void>
```
But after it gets connected to the component, it should just be:
```
(dashboardID: string, variableID: string, value: string) => Promise<void>
```
But somehow the `connect` function from react-redux doesn't understand thunks. The error is
```
  Type '{ onSelectValue: (dashboardID: string, variableID: string, value: string) => (dispatch: any, getState: GetState) => Promise<void>; }' is not assignable to type 'DispatchProps'.
    Types of property 'onSelectValue' are incompatible.
      Type '(dashboardID: string, variableID: string, value: string) => (dispatch: any, getState: GetState) => Promise<void>' is not assignable to type '(contextID: string, variableID: string, value: string) => Promise<void>'.
```
I've never been able to figure this out, do you know how to do this?